libs/orchestration/terraform-azurerm-amido-monitoring/examples/http-logs-metrics/main.tf

```diff
@@ -0,0 +1,51 @@
+data "azurerm_client_config" "current" {}
+
+variable "location" {
+  default = "UK South"
```

Comment:
is this working? should be `uksouth`

Reply:
it works with both, Azure seems to be able to handle it but it does look better with uksouth so that's been changed